File: smiex_api/smiex_api/Models/Incoterm.cs
using System;
using System.Collections.Generic;

namespace smiex_api.Models;

public partial class Incoterm
{
    public int Id { get; set; }

    public int TipusIncontermId { get; set; }

    public int TrackingStepsId { get; set; }

    public virtual ICollection<Oferte> Ofertes { get; set; } = new List<Oferte>();

    public virtual TipusIncoterm TipusInconterm { get; set; } = null!;

    public virtual TrackingStep TrackingSteps { get; set; } = null!;
}

File: simex_app/app/src/main/java/com/example/simex_app/data/models/Incoterm.kt
package com.example.simex_app.data.models

data class Incoterm(
    val id: Int,
    val tipusIncontermId: Int,
    val trackingStepsId: Int,

    val ofertes: List<Oferte> = emptyList(),

    val tipusInconterm: TipusIncoterm? = null,
    val trackingSteps: TrackingStep? = null
)